redditdownload/redditdownload.py

```diff
@@ -161,6 +163,10 @@ def download_from_url(url, dest_file):
     if pathexists(dest_file):
         raise FileExistsException('URL [%s] already downloaded.' % url)
 
+    # use redditupload costum downloader if url match
+    if redditupload.match(url):
```

Comment:
I'll try to add a common code for handling cases like this and then merge this in.